docs/01-about/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+  "label": "About",
+  "collapsible": true,
+  "collapsed": true
```

Comment:
question for curiosity - what happens if one/both of these aren't present? is the category permanently expanded by default? 🤔

Reply:
Looks like both are true by default, so I'll remove them.